terraform/deployments/govuk-test/main.tf

```
@@ -30,6 +30,10 @@ data "aws_security_group" "redis" {
   name = "govuk_backend-redis_access"
 }
 
+data "aws_security_group" "govuk_mongo_access" {
```

nit: If we follow the convention of the ones we have so far, the logical name for this data resource would be "mongodb".

```suggestion
data "aws_security_group" "mongodb" {
```

---

I based it on `govuk_management_access`, damn! Nest time.